sdk/examples/xo_python/sawtooth_xo/xo_cli.py

```diff
@@ -139,6 +139,16 @@ def create_parent_parser(prog_name):
         action='count',
         help='enable more verbose output')
 
+    parent_parser.add_argument(
+        '--auth_user',
+        type=str,
+        help='username to authorize request')
+
+    parent_parser.add_argument(
+        '--auth_password',
+        type=str,
+        help='password to authorize request')
```

Comment:
Same. This is only for if the REST API is using the Basic Auth authorization scheme.